lumispy/utils/__init__.py

```diff
@@ -27,6 +27,386 @@
     axis2invcm,
     data2invcm,
     var2invcm,
-    join_spectra,
     solve_grating_equation,
 )
+
```

Comment:
The utilities should not be in the `__init__.py` file! Create a file `signal.py` instead?